apps/chat/channels.py

```diff
@@ -472,6 +475,11 @@ def send_text_to_user(self, text: str):
     def get_chat_id_from_message(self, message):
         return message.chat_id
 
+    @property
+    def voice_replies_supported(self) -> bool:
+        # TODO: Update turn-python library to support this
```

Comment:
is this TODO still relevant

Reply:
I'm going to merge the other PR into this one, then this TODO wouldn't be relevant anymore

Reply:
I'm going to merge the other PR into this one, then this TODO wouldn't be relevant anymore